keyboards/crkbd/keymaps/madhatter/keymap.c

```diff
@@ -185,6 +182,11 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   }
 
   switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
```

Comment:
```suggestion
    case QWERTY:
      if (record->event.pressed) {
       set_single_persistent_default_layer(_QWERTY);
      }
      break;
```

Also, because you have no other default layers, this is completely unneeded and could be safely removed.